alacritty/src/display.rs

```diff
@@ -207,8 +209,19 @@ impl Display {
         });
 
         // We should call `clear` when window is offscreen, so when `window.show()` happens it
-        // would be with background color instead of uninitialized surface.
-        window.swap_buffers();
+        // would be with background color instead of uninitialized surface. On Wayland we can
+        // safely ignore this call, since the window isn't visible until you actually draw
+        // something into it.
```

Comment:
Let's move the `On Wayland [...]` part to the `if event_loop.is_x11()`, since that reduces the information density of this comment a bit and it's not relevant to the other codepaths anyways.